crates/bevy_render/src/renderer/mod.rs

```diff
@@ -97,13 +101,29 @@ pub struct RenderInstance(pub Instance);
 #[derive(Resource, Clone, Deref, DerefMut)]
 pub struct RenderAdapterInfo(pub AdapterInfo);
 
+/// The texture format used by the renderer.
```

Comment:
Same as above; this doc comment is fairly circular. Try to briefly explain what a texture format is :)

Reply:
Done.